modules/n1ql/pages/n1ql-language-reference/objectfun.adoc

```diff
@@ -1355,6 +1460,99 @@ LIMIT 1;
 ----
 ====
 
+[[fn-obj-type,OBJECT_TYPES()]]
+== OBJECT_TYPES(`expression`)
+
+=== Description
+
+This function takes an object and returns a new one where all the values are replaced with their data types as reported by the TYPE() function.
```

Comment:
Tweaking the wording slightly so that the description (hopefully) explains why you would use the function. Added link to TYPES() function.

```suggestion
This function returns the data type of every field in the supplied object, as reported by the xref:n1ql-language-reference/typefun.adoc#fn-type-type[TYPE()] function.
```